src/components/interfaces/HMI_API.xml

```diff
@@ -3407,7 +3407,7 @@
   </function>
   <function name="GetWayPoints" functionID="GetWayPointsID" messagetype="request">
     <description>Request for getting waypoint/destination data.</description>
-    <param name="wayPointType" type="Common.WayPointType" defvalue="ALL" mandatory="false">
+    <param name="wayPointType" type="Common.WayPointType" defvalue="ALL" mandatory="true">
       <description>To request for either the destination only or for all waypoints including destination</description>
     </param>
```

Comment:
You need to change version according with https://adc.luxoft.com/confluence/display/APPLINK/API+xml+versioning+approach 
and please add business analyst